coder/terraform-provider-coder
# chore: add integration test for provider

This PR adds integration tests for the provider against an arbitrary version of Coder.

Example usage:
```
CODER_VERSION="v2.10.0-devel-3a9a7d199-amd64" go test -v ./integration
```

Testing methodology:
- Create an ephemeral Coder instance running in a container
- Perform initial setup
- Import a predefined template that creates a single file
- Create a workspace that populates the file with the actual output
- Compare the actual versus expected output
